src/pages/api/update-suspicious-repository.js

```diff
@@ -49,38 +51,43 @@ const clearRemark = async (id) => {
 
 //function for update suspicious repo
 const updateSuspiciousRepo = async (req, res) => {
-  const repoId = req.query.id;
+  let repoIds = req.query.ids.split(",");
+  repoIds = repoIds.map(Number);
   const updatedAt = req.query.updatedAt;
   await validation
     .reviewSchema()
     .validate(
       {
-        repoId: req.query.id,
+        repoIds: repoIds,
         updatedAt: updatedAt,
       },
       { abortEarly: false }
     )
     .then(async () => {
       try {
-        let repo = await Repositories.findOne({ where: { id: repoId } });
-        if (!repo) {
-          res.status(404).json({
-            message: "Repository Not Found For Specified Id",
-          });
-        } else {
-          const updatedRepo = await updateRepo(repoId, updatedAt);
-          if (updatedRepo[1].dataValues.parent_repo_id) {
-            await updateParentRepo(
-              updatedRepo[1].dataValues.parent_repo_id,
-              updatedAt
-            );
-            await clearRemark(updatedRepo[1].dataValues.parent_repo_id);
+        await repoIds.map(async (id) => {
+          let repo = await Repositories.findOne({ where: { id: id } });
+          if (!repo) {
+            res.status(404).json({
+              message: "Repository Not Found For Specified Id",
+            });
+          } else {
+            let comment = req.body;
+            const updatedRepo = await updateRepo(id, updatedAt, comment);
+            if (updatedRepo[1].dataValues.parent_repo_id) {
```

Comment:
updatedRepo[1] why 1st index is referred?

Reply:
because update method return a array of updated row and object